certified-connectors/Telstra SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,720 @@
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "x-request-source",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "Telstra SMS Connector",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "description": "Send SMS/MMS leveraging Australia\u0027s leading mobile network",
+        "x-ms-openai-data": {
+          "openai-enabled": false,
+          "operations": [
+            {
+              "operationId": "SendSMS",
+              "x-ms-require-user-confirmation": true
+            }
+          ]
+        }
+      }
+    },
+    "/virtual-numbers/{virtual-number}/callback": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "to": {
+              "type": "string",
+              "description": "to"
+            },
+            "from": {
+              "type": "string",
+              "description": "from"
+            },
+            "timestamp": {
+              "type": "string",
+              "description": "timestamp"
+            },
+            "messageId": {
+              "type": "string",
+              "description": "messageId"
+            },
+            "messageContent": {
+              "type": "string",
+              "description": "messageContent"
+            }
+          }
+        },
+        "description": "Response"
+      },
+      "post": {
+        "responses": {
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "issue": {
+                        "type": "string",
+                        "description": "issue"
+                      },
+                      "location": {
+                        "type": "string",
+                        "description": "location"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      },
+                      "fieldName": {
+                        "type": "string",
+                        "description": "fieldName"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc. but not default